revolv/project/models.py

```diff
@@ -395,8 +396,16 @@ def is_completed(self):
     def categories(self):
         return [category.title for category in self.category_set.all()]
 
+    @property
+    def donation_levels(self):
+        return DonationLevel.objects.filter(project_id=self.id).values()
```

Comment:
Woah, this works? project_id isn't a field of DonationLevel. At any rate, it would be better to just return self.donationlevel_set, correct?